connectors/sources/github.py

```diff
@@ -1312,10 +1313,20 @@ async def get_access_control(self):
             self._logger.warning("DLS is not enabled. Skipping")
             return
 
-        async for user in self.github_client._fetch_all_members(
-            self.configuration["org_name"]
-        ):
-            yield await self._user_access_control_doc(user=user)
+        async for org_name in self._get_orgs():
+            async for user in self.github_client._fetch_all_members(org_name):
```

Comment:
A user may be a member of multiple organizations, and the same user will be indexed multiple times. We can filter out duplicate users by having a set but I'd like to see if there is any reported issue first before we proceed.

Reply:
What's the id for the user? Will it be resolved naturally somehow (e.g. we upload user multiple times, but since ids collide it's okay)

Reply:
The user id from GitHub.

If the same user belongs to multiple organizations, it will be indexed multiple times but it will only result in one document in ES. Adding a set will eliminate the multiple indexing of the same user.